web3/ethtypes.nim

```diff
@@ -122,31 +122,39 @@ type
     dataGasUsed*: Option[Quantity]    # EIP-4844
     excessDataGas*: Option[Quantity]  # EIP-4844
 
+  WithdrawalObject = object
```

Comment:
This is the "normalized" version, `WithdrawalV1` is locked to what's used in engine v1. If fields are added in the future, this one gets extended with `Option`, similar to `BlockObject` / `TransactionObject`.